src/components/JoinUs/OpenSource.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import styled from 'styled-components'
+import getYoutubeId from 'get-youtube-id'
+import { Padding, Margin } from 'styled-components-spacing'
+import { Row, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+import playIcon from '../../images/button-play-default.svg'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColumn,
+  Separator,
+  List,
+  TitleAndBody
+} from './elements'
+
+const MAX_VIDEOS = 3
+
+const getThumbnail = videoId =>
+  `https://img.youtube.com/vi/${videoId}/maxresdefault.jpg`
+
+const TalkLink = styled.a`
```

Comment:
This whole `anchor` will be replaced with the  'soon to be created' StandaloneVideoLink... So I don't really know if it's worth it..

Reply:
this a `display: block` item, so I don't see how they'd fit inside a paragraph. This is basically the anchor with a border, used on each talk link. But this will changed and will be part of the `StandaloneVideoLink`, that's yet to be developed.

Don't think there's much to do here, then, what do you think?

Reply:
closing this, hopefully clarified on the `StandaloneVideoLink`